data/map.txt

```diff
@@ -27213,6 +27213,14 @@ planet "Fenrir Station"
 		threshold 3000
 		fleet "Small Deep Security" 14
 
+planet Fertriery
+	attributes "requires: gaslining"
+	landscape land/nasa8
+	description `Fertriery is a fairly typical world with a particularly dense atmosphere with significant levels of chlorine and fluorine. The only unusual factor is the above-average temperatures, likely due to the decreased energy loss into the surrounding space.`
```

Comment:
```suggestion
	description `Fertriery is a fairly typical world with a particularly dense atmosphere that includes significant levels of chlorine and fluorine. The only unusual factor is the above-average temperatures, likely due to the decreased energy loss into the surrounding space.`
```
Just trying to avoid the double usage of `with`.